Sources/idt/idt.cc

```diff
@@ -183,8 +208,14 @@ class visitor : public clang::RecursiveASTVisitor<visitor> {
   }
 
   clang::DiagnosticBuilder
-  unexported_public_interface(clang::SourceLocation location) {
+  unexported_public_interface(const clang::Decl *D) {
+    return unexported_public_interface(D, get_location(D));
+  }
+
+  clang::DiagnosticBuilder
+  unexported_public_interface(const clang::Decl *D, clang::SourceLocation location) {
     add_missing_include(location);
+    exported_decls_.insert(D);
```

Comment:
This might be valuable to explain - why do we track unexported interfaces as exported after we emit a FixIt?